opba-protocols/xs2a-protocol/src/test/java/de/adorsys/opba/protocol/xs2a/service/xs2a/consent/RequestScopedStub.java

```diff
@@ -37,6 +38,11 @@ public TransientStorage transientStorage() {
         return transientStorage;
     }
 
+    @Override
+    public IgnoreFieldsLoader ignoreFieldsLoader() {
```

Comment:
`FieldsToIgnoreLoader` is more understandable class name compared to `IgnoreFieldsLoader`